configs/Asus/TUF-GAMING-X570-PLUS.conf

```diff
@@ -0,0 +1,88 @@
+chip "nct6798-*"
+# nct6776 values for Asus TUF GAMING X570-PLUS (WI-FI)
+# I'm not entirely sure everything here is correct since no HW monitoring application under Windows shows
+# all the names for all the found sensors. Some of the names and formulas are best guesstimates and could be wrong.
+# You will have to unignore fans and intrusion if you want them to be monitored - I don't have them, I've hidden them
+
+label in0 "Vcore"
+    set in0_min 0.2
+    set in0_max 1.5
+
+# in1
```

Comment:
So, what is `in1`? It's odd that you set min/max values but say nothing else about the inputs.